data-service/server/src/api.js

```diff
@@ -3,24 +3,77 @@ import fastify from 'fastify';
 import fastifyCors from 'fastify-cors';
 import { typeDefs } from './schema';
 import { DgraphDataSource, RxNavDataSource } from './data';
-import { resolvers } from './resolvers'
+import { resolvers } from './resolvers';
+import jwt from 'jsonwebtoken';
+// import fetch from 'node-fetch';
+
+const AUTH_BASE_URL = 'http://127.0.0.1:9990/auth/realms/unified-codes';
+const PUBLIC_KEY = `-----BEGIN PUBLIC KEY-----
```

Comment:
Wonder if we could use some sort of local storage/caching for this? I think we have a registered key pair for our domain, could we use that?

Reply:
Created #96 - worried that this PR is getting too big for its boots.